### ISSUE TITLE
Yet another type inference issue

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#333]
Originally reported by: **Antony Lee (BitBucket: [anntzer](http://bitbucket.org/anntzer), GitHub: @anntzer?)**

---

Not sure what pylint's type inferencer is doing (`cKDTree` is written in an extension module) but...

```py
from scipy.spatial import cKDTree
print(cKDTree([[0, 0]]).data.shape)
```

Pylint thinks that `Class 'data' has no 'shape' member (no-member)` at line 2.

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/333

**Comment by clavedeluna** [comment]
Unless this requires some sort of special config, I'm not able to reproduce the `no-member`  issue.

**Comment by Pierre-Sassoulas** [comment]
I manged to reproduce this on july the fourth, so it probably need scipy installed if I had to guess.

**Comment by clavedeluna** [comment]
> I manged to reproduce this on july the fourth, so it probably need scipy installed if I had to guess.

I indeed have scipy installed

```
(venv) ~/pylint (main) $ pip list | grep scipy
scipy                 1.9.3

(venv) ~/pylint (main) $ cat test.py
from scipy.spatial import cKDTree

(venv) ~/pylint (main) $ pylint test.py

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)
```

**Comment by Pierre-Sassoulas** [comment]
My bad I misread the label I added. I just triaged it and added the "need reproduction" label. Since this july triage frenzy took place we had 100 issues opened and we're back at 650+, it's a never ending thankless task I'm glad you're helping 👍

**Comment by clavedeluna** [comment]
@Pierre-Sassoulas haha yes this is why I'm going thru the issues, 650 is giving me anxiety!
